Marlin/src/lcd/extui/lib/anycubic_chiron/FileNavigator.cpp

```diff
@@ -110,11 +110,12 @@ namespace Anycubic {
       //TFTSer.print(currentfoldername);
       TFTSer.println(filelist.shortFilename());
       TFTSer.print(filelist.shortFilename());
-      TFTSer.println("/");
+      TFTSer.write('/');
```

Comment:
Can you try without this line ? I'm almost sure it's working fine (tried here on my board).